go/libraries/doltcore/sqle/sqlfmt/schema_fmt.go

```diff
@@ -201,6 +201,10 @@ func GenerateCreateTableColumnDefinition(col schema.Column, tableCollation sql.C
 func GenerateCreateTableIndentedColumnDefinition(col schema.Column, tableCollation sql.CollationID) string {
```

Comment:
What is the context of this function being called? Does it work properly for double-quoted default values?

Reply:
afaict, this is being used to generate the schema diffs for `dolt conflicts cat`.